src/invocable.rs

```diff
@@ -345,6 +347,20 @@ macro_rules! method {
 /// ```
 #[macro_export]
 macro_rules! call {
+    ($cls_name:literal :: $fn_name:literal ($( $args:expr ),*) -> $rett:ty) => {
+        (|| {
+            $crate::RttiSystem::get()
+                .get_class($crate::types::CName::new($cls_name))
+                .ok_or($crate::InvokeError::ClassNotFound($cls_name))?
+                .static_methods()
+                .iter()
+                .find(|x| x.as_function().name() == $crate::types::CName::new($fn_name)
+                || x.as_function().short_name() == $crate::types::CName::new($fn_name))
+                .ok_or($crate::InvokeError::FunctionNotFound($fn_name))?
+                .as_function()
+                .execute::<_, $rett>(None, ($( $crate::IntoRepr::into_repr($args), )*))
```

Comment:
```suggestion
                .execute::<_, $rett>(None, ($( $crate::IntoRepr::into_repr($args), )*))
```
red4ext passes a context for static methods: https://github.com/WopsS/RED4ext.SDK/blob/109d7fb0044f86b90d6ccc71f2c30c8a5b6ca208/include/RED4ext/Scripting/Utils-inl.hpp#L54-L58
not sure when and if it's necessary but it's worth checking

Reply:
as we discussed, let's leave it for now and fix it if problem arises.